setup.py

```diff
@@ -308,7 +308,7 @@ def geometry_extensions():
     install_requires=[
         "numpy>=1.23,<2.0.0a0",
         "scipy",
-        "astunparse",
+        "astunparse;python_version<'3.9'",
```

Comment:
```suggestion
```
I don't think this is worth the hassle, we only say we support SPEC 0 anyway and that's long since dropped 3.8 https://scientific-python.org/specs/spec-0000/